sklearn/feature_extraction/text.py

```diff
@@ -936,6 +951,8 @@ def transform(self, raw_documents):
                 "Iterable over raw text documents expected, "
                 "string object received.")
 
+        self._check_ngram_range()
```

Comment:
This can be removed. Checking in fit is sufficient, transform here will error anyway if fit wasn't previously called.